RHEL, Rocky , Alma Linux - Build rpms for versions 8, 9, 10

**Is your feature request related to a problem? Please describe.**
Package rpm version 2.5.1 for RHEL based linux systems. The listed dependencies do not seem to exist via the dnf command for versions 8, 9 or 10 

**Describe the solution you'd like**
Builds fail because I am unable to identify the right dependencies.

**Describe alternatives you've considered**
I cannot find these dependencies in the dnf ..
    libxml-2.0 2.6.16 or later
    icu4c 4.8 or later

and I get this error
`checking for pkg-config... /usr/bin/pkg-config
checking for genrb... no
configure: error: genrb was not found`

IBM offer 2.4 for RHEL but the ltfs_ordered_copy function does not work due to dependencies. Your 2.5.1 ltfs_ordered_copy works but it would be great to have all installation in 2.5.1

**Additional context**
[Add any other context or screenshots about the feature request here.](https://forums.rockylinux.org/t/ltfs-build-for-rocky-8-10-rocky-9/20056)

--- a/.devcontainer/rocky9/Dockerfile
+++ b/.devcontainer/rocky9/Dockerfile
@@ -1,0 +1,54 @@
+FROM rockylinux:9
+
+# Install EPEL repository for additional packages
+RUN dnf install -y epel-release && \
+    dnf update -y
+
+# Enable CRB (CodeReady Builder) repository for development packages
+RUN dnf install -y dnf-plugins-core && \
+    dnf config-manager --set-enabled crb
+
+# Install build dependencies for LTFS
+RUN dnf install -y \
+    autoconf \
+    automake \
+    libtool \
+    make \
+    gcc \
+    pkg-config \
+    fuse-devel \
+    fuse \
+    libxml2-devel \
+    libicu-devel \
+    icu \
+    libuuid-devel \
+    net-snmp-devel \
+    diffutils
+
+# Install development and debugging tools
+RUN dnf install -y \
+    gdb \
+    valgrind \
+    git \
+    sudo
+
+# Install GitHub CLI
+RUN dnf install -y 'dnf-command(config-manager)' && \
+    dnf config-manager --add-repo https://cli.github.com/packages/rpm/gh-cli.repo && \
+    dnf install -y gh
+
+# Clean up dnf cache
+RUN dnf clean all
+
+# Create non-root user
+ARG USERNAME=vscode
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+RUN groupadd --gid $USER_GID $USERNAME && \
+    useradd --uid $USER_UID --gid $USER_GID -m $USERNAME && \
+    echo "$USERNAME ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers.d/$USERNAME && \
+    chmod 0440 /etc/sudoers.d/$USERNAME
+
+USER $USERNAME
+WORKDIR /workspaces

--- a/.devcontainer/rocky9/devcontainer.json
+++ b/.devcontainer/rocky9/devcontainer.json
@@ -1,0 +1,18 @@
+{
+    "name": "LTFS Development (Rocky Linux 9)",
+    "build": {
+        "dockerfile": "Dockerfile",
+        "context": "../.."
+    },
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "ms-vscode.cpptools",
+                "ms-vscode.cmake-tools",
+                "ms-vscode.makefile-tools"
+            ]
+        }
+    },
+    "postCreateCommand": "./autogen.sh && ./configure --prefix=/workspaces/ltfs-oss",
+    "remoteUser": "vscode"
+}

--- a/.devcontainer/ubuntu2404/Dockerfile
+++ b/.devcontainer/ubuntu2404/Dockerfile
@@ -1,0 +1,43 @@
+FROM ubuntu:24.04
+
+# Avoid prompts from apt
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Update and install build dependencies for LTFS
+RUN apt-get update && apt-get install -y \
+    autoconf \
+    automake \
+    libtool \
+    make \
+    gcc \
+    pkg-config \
+    libfuse-dev \
+    fuse \
+    libxml2-dev \
+    libicu-dev \
+    uuid-dev \
+    libsnmp-dev \
+    icu-devtools
+
+# Install development and debugging tools
+RUN apt-get install -y \
+    gdb \
+    valgrind \
+    git \
+    sudo
+
+# Clean up apt cache
+RUN apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# Create non-root user
+ARG USERNAME=vscode
+ARG USER_UID=1001
+ARG USER_GID=$USER_UID
+
+RUN groupadd --gid $USER_GID $USERNAME && \
+    useradd --uid $USER_UID --gid $USER_GID -m $USERNAME && \
+    echo "$USERNAME ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers.d/$USERNAME && \
+    chmod 0440 /etc/sudoers.d/$USERNAME
+
+USER $USERNAME
+WORKDIR /workspaces

--- a/.devcontainer/ubuntu2404/devcontainer.json
+++ b/.devcontainer/ubuntu2404/devcontainer.json
@@ -1,0 +1,21 @@
+{
+    "name": "LTFS Development (Ubuntu 24.04)",
+    "build": {
+        "dockerfile": "Dockerfile",
+        "context": "../.."
+    },
+    "features": {
+        "ghcr.io/devcontainers/features/github-cli:1": {}
+    },
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "ms-vscode.cpptools",
+                "ms-vscode.cmake-tools",
+                "ms-vscode.makefile-tools"
+            ]
+        }
+    },
+    "postCreateCommand": "./autogen.sh && ./configure --prefix=/workspaces/ltfs-oss",
+    "remoteUser": "vscode"
+}

--- a/.gitignore
+++ b/.gitignore
@@ -23,6 +23,8 @@ ltfs.pc
 .libs/
 messages/.lib
 .dirstamp
+# local settings for claude code
+.claude
 # Files created by OS
 .DS_Store
 # Files for development

--- a/AGENTS.md
+++ b/AGENTS.md
@@ -1,6 +1,6 @@
-# CLAUDE.md
+# AGENTS.md
 
-This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.
+This file provides guidance to AI agents when working with code in this repository.
 
 ## Project Overview
 

